data/actions/scripts/other/potions.lua

```diff
@@ -27,132 +72,90 @@ function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	if target == nil or not target:isPlayer() then
 		return true
 	end
-
+	
 	if player:getCondition(CONDITION_EXHAUST_HEAL) then
 		player:sendTextMessage(MESSAGE_STATUS_SMALL, Game.getReturnMessage(RETURNVALUE_YOUAREEXHAUSTED))
 		return true
 	end
 
-	local itemId = item:getId()
-	if itemId == antidotePot then
-		if not antidote:execute(target, numberToVariant(target:getId())) then
-			return false
-		end
-
+	local stats = potions[item:getId()]
+	if stats then
+		if (stats.vocations or stats.minLevel) and not player:getGroup():getAccess() then
+			if not stats.minLevel then
+				stats.minLevel = 1
+			end
+			if not stats.vocations then
+				stats.vocations = {}
+			end
+			
+			local vocationList, mvoc = {}, player:getVocation()
+			while mvoc do
```

Comment:
Nvm, it don't hurt us, but i just tried to care with custom servers, i will update as soon as I am on a pc.